prime-angular-frontend/src/app/modules/phsa-eforms/pages/bcsc-demographic/bcsc-demographic.component.html

```diff
@@ -32,7 +32,12 @@
     </section>
 
     <app-contact-information-form [form]="form"
-                                  [showSmsPhone]="false"></app-contact-information-form>
+                                  [showSmsPhone]="false">
+      <div notificationInfoSummary>
```

Comment:
> When you use something like this it makes it look like an Angular directive when it isn't. You can create a directive like we do for components like `PageSubheader2Component` where you might add `ContactInfoSummaryDirective` and be able to drop the `<div>` since it isn't a structural or styled element:
> 
> ```js
> <ng-container appContactInfoSummary>Lorem ipsum dolor sit...</ng-container>
> ```
> 
> In the `ContactInformationForm` you can do a similar `select="[appContactInfoSummary]"`, but the directive allows you in the controller to use `@ContentChildren(ContactInfoSummaryDirective) public summary QueryList<ContactInfoSummaryDirective>;` to check the length of the projected content and reduce a bit more of the markup using `NgIfElse` structural directive on a getter like `<ng-container *ngIf="hasCustomSummary; else default">...</ng-container>` and have the default in a `<ng-template #default>...</ng-template>`.

Yep PageSubheader2Component is a very good example.👍